apps/nfdiv/nfdiv-cron-migrate-cases/prod/00.yaml

```diff
@@ -6,6 +6,7 @@ spec:
   releaseName: nfdiv-cron-migrate-cases
   values:
     job:
+      disableActiveClusterCheck: false
```

Comment:
Do we need to create prod.yaml for this cron?

Reply:
Yes I think I missed out on renaming the file to prod.yaml